### PR TITLE
Update zh-CN (Simplified Chinese) translation

### PR DESCRIPTION
**Pull request by ykla**
Update zh-CN (Simplified Chinese) translation.

- The Simplified Chinese translation of Raspberry Pi Imager has been revised to more closely align with the original text.
- Any entries that were not present in the original text have been removed, and any entries that were not translated have been updated.
- The translation has been subjected to rigorous accuracy checks to ensure optimal alignment with the target audience of Raspberry Pi.
- The terminology employed in reference to Apple macOS has been aligned with the Simplified Chinese translations utilized within the macOS system itself.
- Added translation for Linux desktop icon file

Note: I am unfamiliar with the programming environment on macOS and the process of modifying `Info.plist.in`.  **Therefore, I request that someone with the requisite expertise perform these modifications in the future. I am not familiar with Windows at all `rpi-imager.rc` or `rpi-imager.nsi.in`. Sorry.**

### CODE REVIEW

**Comment by tdewey-rpi** [comment]
@YuzukiTsuru As the last substantial editor of the zh-CN translations, do you have an opinion on this change?

If I receive no feedback within a week, I will use machine translation to verify with a view to merging.

**Comment by tdewey-rpi** [comment]
Thanks for your thoroughness, @ykla.

As I've had no feedback from the previous translator, I will proceed with a view to accepting this change. As I'm not proficient in the language, this will involve a review with machine translation.

I'll investigate the changes for macOS, but the Windows installer may lag behind until someone with experience of localising such things volunteers their time.

File: src/i18n/rpi-imager_zh.ts [hunk 1]
```diff
@@ -7,89 +7,85 @@
...
         <location filename="../downloadthread.cpp" line="187"/>
         <source>Error removing existing partitions</source>
-        <translation>删除现有分区时出错</translation>
+        <translation>删除现有分区时发生错误</translation>
     </message>
     <message>
         <location filename="../downloadthread.cpp" line="213"/>
         <source>Authentication cancelled</source>
-        <translation>认证已取消</translation>
+        <translation>验证已取消</translation>
```
Comment (tdewey-rpi) [inline src/i18n/rpi-imager_zh.ts:53]: Machine translation turned this into "Verification cancelled". This message is, I believe, displayed in response to a macOS authentication window cancel action - have you seen this translation used in similar cases, @ykla?

Comment (ykla) [reply src/i18n/rpi-imager_zh.ts:53]: The term "Two Factor Authentication," or 2FA, is translated as 双因素身份验证 in Chinese. In fact, in Chinese, "Authentication" and "Verification" share the same meaning. See also <https://www.microsoft.com/zh-cn/security/business/security-101/what-is-two-factor-authentication-2fa>.

Comment (tdewey-rpi) [reply src/i18n/rpi-imager_zh.ts:53]: Thanks for the explanation, @ykla. Every day is a learning experience!

**Review by tdewey-rpi — APPROVED** [review]
Outstanding contribution, @ykla, and very much appreciated. I particularly applaud your attention to the untranslated new strings, and to maintaining the tone of the tool.

**Pull request merged**